smeargle.py

```diff
@@ -48,6 +52,9 @@ def sd_api_call():
     # Increment the total api call counter
     api_call_count += 1
 
+    if dry_run_enabled:

```

Comment:
This way you don't have to increment the api call outside the function if its a dry run